src/components/StakingTimeline/index.tsx

```diff
@@ -240,11 +224,8 @@ const StakingTimeline: FC<{ statuses?: FlowStepStatus[] } & StackProps> = ({
         >
           You will need to run a node for applications that you have authorized
           to earn rewards. If you don’t have one, learn how to do it{" "}
-          <InternalLink to={APPLICATION_DOCS_URL}>here</InternalLink> or contact
-          a{" "}
-          <InternalLink to={STAKING_PROVIDER_URL}>
-            Staking Provider.
-          </InternalLink>
+          <Link to={APPLICATION_DOCS_URL}>here</Link> or contact a{" "}
+          <Link to={STAKING_PROVIDER_URL}>Staking Provider.</Link>
```

Comment:
Non-blocking:

Let's move the dot outside of `Link` component:

```suggestion
          <Link to={STAKING_PROVIDER_URL}>Staking Provider</Link>.
```